### DISCUSSION TITLE
Fuzzylist lets unicode domains slip through

### DISCUSSION THREAD

**Discussion by sime** [MetaMask/eth-phishing-detect#11728] [Ideas]
Even though [trezor.io](https://github.com/MetaMask/eth-phishing-detect/blob/main/src/config.json#L21) is on the fuzzylist of eth-phishing-detect, domains like trezór(.)net (xn--trezr-3ta(.)net) still get through.

Can fuzzylist be extended to test unicode domains?

**Comment by legobeat** [comment]
Related:

#9407

https://github.com/MetaMask/phishing-warning/issues/45
